src/cli/utils.rs

```diff
@@ -0,0 +1,160 @@
+use std::{
+    fs::File,
+    io::Write,
+    path::{Path, PathBuf},
+};
+
+use figment::{
+    providers::{Format, Toml},
+    Figment,
+};
+use miden_client::{
+    client::{rpc::NodeRpcClient, Client},
+    config::ClientConfig,
+    store::Store,
+};
+use miden_objects::{
+    accounts::AccountId,
+    crypto::rand::FeltRng,
+    notes::{NoteExecutionHint, NoteTag, NoteType},
+    NoteError,
+};
+use miden_tx::TransactionAuthenticator;
+
+use super::{get_account_with_id_prefix, CLIENT_CONFIG_FILE_NAME};
+use crate::cli::info;
+
+/// Returns an Account ID string or the default one defined in the Client config
```

Comment:
How about something like:

```suggestion
/// Returns a tracked Account ID matching a hex string or the default one defined in the Client config
```